packages/tkg-clusterclass-azure/bundle/config/upstream/base.yaml

```diff
@@ -1395,4 +1408,15 @@ spec:
             cloud-provider: azure
           name: '{{ ds.meta_data["local_hostname"] }}'
       useExperimentalRetryJoin: true
-      preKubeadmCommands: []
+      preKubeadmCommands:
+      - echo "running preKubeadmCommands…"
+---
+apiVersion: v1
```

Comment:
Even as a hack/workaround, this probably should not be part of base.yaml. Augment the gist to incorporate this instead?